api/modules/users/views.py

```diff
@@ -223,3 +223,29 @@ def remove_user_status(request):
         return Response(str(e), status=status.HTTP_400_BAD_REQUEST)
 
     return Response(status=status.HTTP_200_OK)
+
+
+@api_view(['POST'])
+def update_user_password(request):
+
+    old_password = request.POST.get('old_password', None)
+    new_password = request.POST.get('new_password', None)
+
+    user = request.user
+    confirm_password = user.password
+
+    if not old_password or not new_password:
+        error_message = "Missing parameters in request"
+        return Response(error_message, status=status.HTTP_400_BAD_REQUEST)
+
+    if not confirm_password == old_password:
```

Comment:
I'm entering my correct old password still I am getting `Password not found`
I printed my `confirm_password` which was 
```
pbkdf2_sha256$100000$pyGP7ZlnqJsS$sbB5oNL3EvQJ+EBwSdcK26ttr4kHhx/ypjLCq7Sg1oQ=
```
I don't think a string matching would be good enough here

Reply:
should I then do regex matching?

Reply:
Please read online and try to figure out
And test the changes locally yourself before you update the PR